infra/params.dev.json

```diff
@@ -71,6 +71,12 @@
         "enableSearch": {
             "value": true
         },
+        "enableMonitoring": {
```

Comment:
Why is enableMonitoring set in dev, but in prod and test enableObservability is set?